packages/tabs/tabs.scss

```diff
@@ -80,6 +80,10 @@
     text-decoration: none;
     white-space: nowrap;
 
+    @include jkl.small-device {
+        padding: #{jkl.$spacing-8} #{jkl.$spacing-24} #{jkl.$spacing-12} #{jkl.$spacing-2};
+    }
```

Comment:
Dette vil vel ende med å øke noe av paddingen som er satt for compact-mode?
https://github.com/fremtind/jokul/blob/bcdc1154f4d29d955295f2b6e3c0d27db9d1371b/packages/tabs/tabs.scss#L27

Reply:
Padding på compact for mobil var 32px, men skal også nå være på 24px (likt som på default) :)

Reply:
Ah ok, jeg syns det så ut som et par av de andre verdiene økte litt men jeg mappet sikkert bare feil mellom xs/s/m og tallskalaen.